### DISCUSSION TITLE
Can you non-destructively switch to a different scsi driver?

### DISCUSSION THREAD

**Discussion by dburr** [erichelgeson/BlueSCSI-v1#100] [Q&A]
I have a lot of image files that were created using the LIDO driver, and after the recent revelations about performance (or lack thereof) of the LIDO Driver, I'd like to switch them to either the Apple or FWB drivers. Is there any way to do this non-destructively, or do I need to create new image files and copy the contents from one to the other in the Finder, etc.?

**Comment by erichelgeson** [comment]
Sure, formatting utilities have a way to just update drivers. Find & click the Update / Update driver button and reboot. Always take a backup just incase.
